test_runner/performance/pgvector/README.md

```diff
@@ -1,3 +1,20 @@
+# Source of the dataset for pgvector tests
+
+This readme was copied from https://huggingface.co/datasets/Qdrant/dbpedia-entities-openai3-text-embedding-3-large-1536-1M
```

Comment:
Different URL is still mentioned at the bottom of the file. It's the one I used.

Reply:
removed it now